js/main.js

```diff
@@ -224,6 +224,11 @@ const MM = (function () {
 	 * @param {object} [options] Optional settings for the hide method.
 	 */
 	const hideModule = function (module, speed, callback, options = {}) {
+		// reset lockString and prepare for set the new one
+		if (options.reset === true && options.lockString) {
```

Comment:
is checking for options.lockString necessary?

Reply:
yes, because lockString replacement
reset and lockString works both (security)

Reply:
I don't really see the point of destroying the Array without placing a new lock
but ... I may be wrong in my way of seeing things

@rejas: What do you think ?